src/bayesian_linear_regression.jl

```diff
@@ -74,6 +74,36 @@ function __compute_inference_quantities(fx::FiniteBLR, y::AbstractVector{<:Real}
     return Uw, Bt, δy, logpdf_δy, Λεy
 end
 
+#
+# Input type: RowVecs
+#
+
+const FiniteBLRRowVecs = FiniteGP{<:BayesianLinearRegressor,<:RowVecs}
+
+finite_blr(fx::FiniteBLRRowVecs) = FiniteGP(fx.f, ColVecs(fx.x.X'), fx.Σy)
```

Comment:
Another way to do this would be to have something like
```
x_as_colvecs(fx::FiniteBLR) = fx.x.X
x_as_colvecs(fx::FiniteBLRRowVecs) = ColVecs(fx.x.X')
```
and then rewrite the `FiniteBLR` methods above to use `x_as_colvecs` where they currently use `fx.x.X` - would obviate the need to have the methods below (and would also not need to recreate the `FiniteGP` every time) - what do you think?

Reply:
🤦 why didn't I think of that! A much better solution, will implement now.